syft/execution/action.py

```diff
@@ -4,7 +4,7 @@
 import syft as sy
 from syft.execution.placeholder import PlaceHolder
 from syft.execution.placeholder_id import PlaceholderId
-from syft.serde.syft_serializable import SyftSerializable
+from syft.generic.abstract.syft_serializable import SyftSerializable
```

Comment:
Q: I found the ```syft_serializable``` file in ```serde``` folder.
Hm...how is this working?

Reply:
oh! it was moved!
Actually I had to move it because of very nasty circular imports -> generic.abstract sounded like the bestplace

Reply:
@gmuraru do you think this is ok?

Reply:
Ohh..I did not see the files being moved!